README.md

```diff
@@ -87,20 +88,27 @@ dotnet add package Microsoft.Agents.AI
 Create a simple Azure Responses Agent that writes a haiku about the Microsoft Agent Framework
 
 ```python
-# pip install agent-framework
+# pip install agent-framework --prerelease=allow
 # Use `az login` to authenticate with Azure CLI
 import asyncio
 from agent_framework.azure import AzureOpenAIResponsesClient
 from azure.identity import AzureCliCredential
 
+
 async def main():
-  # Initialize a chat agent with basic instructions
-  agent = AzureOpenAIResponsesClient(credential=AzureCliCredential()).create_agent(
-    name="HaikuBot",
-    instructions="You are an upbeat assistant that writes beautifully.",
-  )
+    # Initialize a chat agent with Azure OpenAI Responses
+    agent = AzureOpenAIResponsesClient(
+        endpoint="https://your-custom-endpoint.openai.azure.com/",
```

Comment:
These are automatically picked up from env vars or a .env file, which is why they aren’t explicitly included.